src/megaclient.cpp

```diff
@@ -6008,8 +6008,9 @@ error MegaClient::invite(const char* email, visibility_t show)
  * Attributes are stored as base64-encoded binary blobs. They use internal
  * attribute name prefixes:
  *
- * "*" - Private and CBC-encrypted.
- * "+" - Public and plain text.
+ * "*" - Private and CBC-encrypted. Use a TLV container (key-value)
```

Comment:
AES-CCM or -GCM encrypted, not CBC (cipher block chaining is used for other simple keys only, but user attributes use CCM or GCM)